langfuse/callback/langchain.py

```diff
@@ -743,6 +743,9 @@ def _update_trace(self, run_id: str, parent_run_id: Optional[str], output: any):
         ):
             self.trace = self.trace.update(output=output)
 
+        # Remove the run details after updating the trace.
```

Comment:
Can we quickly rename the function? E.g. _update_trace_and_remove_state(...) to make this more readable?

Reply:
If the tests complete, i think we are good to go here. Can you add assertions to e.g. https://github.com/noble-varghese/langfuse-python/blob/c1aee1422c48c7cabb4fc4febf7407fca11ccfa4/tests/test_langchain.py#L886 to check that the runs map is empty?

Reply:
Fixed on Issue #527 and added assertions as well.